desktop/src/main/java/bisq/desktop/primary/main/left/LeftNavButton.java

```diff
@@ -104,13 +111,38 @@ class LeftNavButton extends Pane implements Toggle {
         getChildren().add(label);
 
         if (hasSubmenu) {
-            arrowDownIcon = BisqIconButton.createIconButton("nav-arrow-down");
-            arrowDownIcon.setOpacity(0.4);
+            expandIcon = BisqIconButton.createIconButton("nav-arrow-right");
+            collapseIcon = BisqIconButton.createIconButton("nav-arrow-down");
+            expandIcon.setOnMouseClicked(e -> {
+                getIsSubMenuExpanded().set(true);
+                if (expandCollapse != null) {
+                    expandCollapse.accept(this.navigationTarget);
+                }
+            });
+            collapseIcon.setOnMouseClicked(e -> {
+                getIsSubMenuExpanded().set(false);
+                if (expandCollapse != null) {
+                    expandCollapse.accept(this.navigationTarget);
+                }
+            });
 
-            arrowDownIcon.setLayoutX(LeftNavView.EXPANDED_WIDTH - 20);
-            arrowDownIcon.setLayoutY(10);
+            expandCollapseIcon = new VBox();
+            expandCollapseIcon.setOpacity(0.4);
 
-            getChildren().addAll(arrowDownIcon);
+            expandCollapseIcon.setLayoutX(LeftNavView.EXPANDED_WIDTH - 20);
+            expandCollapseIcon.setLayoutY(10);
+            expandCollapseIcon.getChildren().setAll(expandIcon);
+
+            getChildren().addAll(expandCollapseIcon);
+
+            EasyBind.subscribe(getIsSubMenuExpanded(), isExpanded -> {
+                assert expandCollapseIcon != null;
```

Comment:
We use usually guavas `Preconditions.checkArgumnet` or `Objects.nonNull`. Do you have reasons to prefer assert over the other 2? A reason why we dont use assert is because its deactivated by default as they are intended for dev/testing environment.